crowdsec-docs/docs/cti_api/integration_maltego.md

```diff
@@ -0,0 +1,141 @@
+---
+id: integration_maltego
+title: Maltego Transforms
+sidebar_position: 4
+---
+
+Maltego transforms, which allow users to enrich IP entities in maltego with CrowdSec CTI intelligence.
```

Comment:
We could introduce to the following procedures with :  
In order to add a transform to Maltego you'll need to fetch it on a Transform Distribution Server (TDS)  
In this documentation  we'll tell you:
* How to create your private TDS (pTDS)
* Where to install and configure the crowdsec transforms
* What our 11 transforms do